packages/neuron-ui/src/widgets/Dialog/index.tsx

```diff
@@ -17,6 +17,9 @@ interface DialogProps {
   children?: React.ReactChild
   footer?: React.ReactNode
   isLoading?: boolean
+  confirmProps?: object
+  hideConfirm?: boolean
+  hideCancel?: boolean
```

Comment:
`hideConfirm` and `hideCancel` sounds like operations, but not status. Maybe you can update them with `isConfirm` and `isCancel` or something else.